nrf_dm/doc/CHANGELOG.rst

```diff
@@ -12,6 +12,16 @@ All the notable changes to this project are documented in this file.
 main
 ****
 
+Changes
+=======
+* Improved the resilience to interference during synchronization.
```

Comment:
The resilience of what?

Reply:
Trying to say that the synchronization procedure will fail less often due to interference

Reply:
Reworded this a bit

Reply:
> Increased the synchronization procedure's resilience to interference.

Thanks. Updated to this now.